src/containers/List/ListWrapper.tsx

```diff
@@ -105,8 +108,6 @@ const ListWrapper: React.FC = () => {
     }
   }
 
-  if (loading) return <Loading />
-
```

Comment:
This was the problem! Was showing the "Loading" component in place of the whole ListWrapper which was wiping out the Search (and filters). There is already a "Loading" component within the list, so we don't need one here as well.